march_ethercat/Slave.cpp

```diff
@@ -0,0 +1,30 @@
+//
+// Base class source for all slaves
+//
+
+#include "Slave.h"
+
+// Constructor
+Slave::Slave(std::string name, uint16 number)
```

Comment:
What is the purpose of different classes if we are going to refer to them by their type at all times?